everett/manager.py

```diff
@@ -240,6 +241,100 @@ def __repr__(self):
         return '<ListOf(%s)>' % qualname(self.sub_parser)
 
 
+def _parse_rwx(modestr, default):
+    """parse out 'r', 'w' and 'x'.  use 'default' if none used or specified"""
+    mode = 0
+    if 'r' in modestr:
+        mode |= os.R_OK
+    if 'w' in modestr:
+        mode |= os.W_OK
+    if 'x' in modestr:
+        mode |= os.X_OK
+    if not mode:
+        mode = default
+    return mode
+
+
+def _mode2english(mode):
+    """turn a file mode into an english phrase"""
+    perms = []
+    if mode & os.R_OK: perms.append('read')
+    if mode & os.W_OK: perms.append('write')
+    if mode & os.X_OK: perms.append('execute')
+    return (','.join(perms[:-1]) + ' and ' if len(perms)>1 else '') + perms[-1]
+
+
+class PathValue(object):
+    """
+    A parser for paths
+
+    config('Pathname', parser=PathValue('w', isdir=True, exists=False))
```

Comment:
Change `exists` here to `check_exists`. The docstring suggests this turns on and off the check for whether the file exists, but the name suggests that it might have three modes of:

* `True`: verifies it exists
* `False`: verifies it doesn't exist
* `None`: do nothing

Unless you have a compelling use case for needing that, I'd rather go the simpler route.

Reply:
Er, change it here in the docs and also in `__init__`.

Reply:
The simpler route is what's actually implemented, so I changed the name as you suggested.